src/store/wallet/effects/import/import.ts

```diff
@@ -752,6 +764,143 @@ export const migrateKeyAndWallets =
     });
   };
 
+export const deferredImportErrorNotification = (): Effect => async dispatch => {
+  dispatch(dismissOnGoingProcessModal());
+  await sleep(600);
+  dispatch(
+    showBottomNotificationModal({
+      type: 'error',
+      title: t('Problem importing key'),
+      message: t('There was an issue importing your key. Please try again.'),
+      enableBackdropDismiss: false,
+      actions: [
+        {
+          text: t('IMPORT KEY'),
+          action: () => {
+            navigationRef.navigate('Wallet', {screen: 'CreationOptions'});
+          },
+          primary: true,
+        },
+        {
+          text: t('MAYBE LATER'),
+          action: () => {},
+          primary: false,
+        },
+      ],
+    }),
+  );
+};
+
+export const deferredImportMnemonic =
+  (
+    importData: {words?: string; xPrivKey?: string},
+    opts: Partial<KeyOptions>,
+    context?: string,
+    skipNotification?: boolean,
+  ): Effect =>
+  async (dispatch, getState): Promise<void> => {
+    try {
+      const {WALLET} = getState();
+      let _context = context;
+      if (_context !== 'onboarding') {
+        _context = 'deferredImport';
+      }
+      let continueTapped = false;
+
+      if (!skipNotification) {
+        dispatch(
+          showBottomNotificationModal({
+            type: 'wait',
+            title: t('Please wait'),
+            message: t(
+              'Your key is still being imported and will be available shortly. ',
+            ),
+            enableBackdropDismiss: false,
+            actions: [
+              {
+                text: t('GOT IT'),
+                action: () => {
+                  continueTapped = true;
+                  backupRedirect({
+                    context: _context,
+                    navigation: navigationRef,
+                    walletTermsAccepted: WALLET.walletTermsAccepted,
+                  });
+                },
+                primary: true,
+              },
+            ],
+          }),
+        );
+      }
+
+      dispatch(updateDeferredImport({importData, opts}));
+
+      const key = (await dispatch<any>(
+        startImportMnemonic(importData, opts),
+      )) as Key;
+      await dispatch(startGetRates({}));
+      await dispatch(startUpdateAllWalletStatusForKey({key}));
+      await dispatch(updatePortfolioBalance());
+      dispatch(setHomeCarouselConfig({id: key.id, show: true}));
+
+      dispatch(dismissOnGoingProcessModal());
+      await sleep(600);
+      dispatch(clearDeferredImport());
+      dispatch(
+        logSegmentEvent('track', 'Imported Key', {
+          context: context || '',
+          source: 'RecoveryPhrase',
+        }),
+      );
+
+      // TODO: Update language
```

Comment:
Any change needed here ?

Reply:
yes, updated the language to match mocks.